opencti-platform/opencti-front/tests_e2e/model/reportDetails.pageModel.ts

```diff
@@ -18,8 +19,13 @@ export default class ReportDetailsPage {
     return this.page.getByLabel('Update', { exact: true });
   }
 
-  goToOverviewTab() {
-    return this.page.getByRole('tab', { name: 'Overview' }).click();
+  async goToOverviewTab() {
+    await this.page.getByRole('tab', { name: 'Overview' }).click();
+    await expect(this.page.getByTestId('report-overview')).toBeVisible();
```

Comment:
I think this expect should be done in the test and not the page model to not mix concerns. Page model to help manipulating the app, tests for assertions. What do you think?

Reply:
I don't have strong opinion on that it's fine for me, I'm moving.